### PR TITLE
chore: add UnpublishedEntity in storybook (CALCOM-10760)

### PR DESCRIPTION
**Pull request by gitstart-calcom**
## DEMO

https://www.loom.com/share/fbdbea3c65dd4c808b03c839a8734c8a

### CODE REVIEW

**Review by Udit-takkar — COMMENTED** [review]
@gitstart-calcom can we use this addon https://storybook.js.org/addons/storybook-i18next to see the translated text  instead of keys "team_is_unpublished"

**Comment by gitstart-calcom** [comment]
Hi @Udit-takkar, we have added the support for translation. Please take a look

File: apps/storybook/.storybook/i18next.js [hunk 1]
```diff
@@ -1,9 +1,29 @@
 import i18n from "i18next";
 import { initReactI18next } from "react-i18next";
 
+const ns = ["common"];
+const supportedLngs = ["en", "fr"];
```
Comment (Udit-takkar) [inline apps/storybook/.storybook/i18next.js:5]: we support more languages but fine for now

**Review by Udit-takkar — APPROVED** [review]
@gitstart-calcom  LGTM. great work

**Pull request merged**